libraries/standard/mqtt/utest/CMakeLists.txt

```diff
@@ -81,3 +81,20 @@ create_test(${utest_name}
             "${utest_dep_list}"
             "${test_include_directories}"
         )
+
+# create targets for testing mqtt_lightweight.c
+set(utest_name "${project_name}_state_utest")
+set(utest_source "${project_name}_state_utest.c")
+
+set(utest_link_list "")
+list(APPEND utest_link_list
+            -lunity
+            lib${real_name}.a
+        )
+
+create_test(${utest_name}
+            ${utest_source}
+            "${utest_link_list}"
+            "${utest_dep_list}"
+            "${test_include_directories}"
+        )
```

Comment:
minor: please add a new line at the end of the file.

Reply:
Added